.Lib9c.DevExtensions.Tests/Action/ManipulateStateTest.cs

```diff
@@ -503,14 +579,21 @@ public void SetMultipleStates()
             var questData = FetchQuest().Last();
             var targetQuestIdList = (List<int>)questData[0];
             var questList = (QuestList)questData[1];
+            var balanceList = FetchBalance()
+                .Select(objects => (addr: (Address)objects[0], fav: (FungibleAssetValue)objects[1]))
+                .Where(tuple => !(tuple.fav.Currency.Ticker == "NCG" && tuple.fav.MajorUnit > 100))
```

Comment:
Should we ignore balance more than 100 NCG?

Reply:
The return values of `FetchBalance()` method has the `FungibleAssetValue` which contains the maximum quantity of `NCG` the `GoldBalanceState` can has.😅
So this line avoid that because `IAccountStateDelta.TransferAsset()` can throw an exception.